packages/trader/src/AppV2/Components/SymbolSearchResults/symbol-search-results.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import SymbolNotFound from '../SymbolNotFound';
+import { useGetSymbolSearchResults } from 'AppV2/Hooks/useGetSymbolSearchResults';
+import { observer } from '@deriv/stores';
+import MarketCategoryItem from '../MarketCategoryItem';
+
+type TSymbolSearchResults = {
+    searchValue: string;
+    setSearchValue: (input: string) => void;
+    setIsOpen: (input: boolean) => void;
+    setSelectedSymbol: (input: string) => void;
+};
+const SymbolSearchResults = observer(({ searchValue, setIsOpen, setSelectedSymbol }: TSymbolSearchResults) => {
+    const searchResults = useGetSymbolSearchResults(searchValue);
+
+    return (
+        <div className='symbol-search-results__container'>
+            {searchValue === '' && (
```

Comment:
May a ternary will be better here? Like `searchValue ? <Another option>: <Try searching for markets or keywords> `

Reply:
i did it this way because there'll be a nested ternary when searchValue !== ''